src/early-errors.js

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-import reduce, { MonoidalReducer } from 'shift-reducer';
-import { isStrictModeReservedWord } from './utils';
-import { ErrorMessages } from './errors';
+const { default: reduce, MonoidalReducer } = require('shift-reducer');
```

Comment:
```suggestion
const { reduce, MonoidalReducer } = require('shift-reducer');
```

(It's available under both names, so might as well use the one we want.)